explore_utils.py

```diff
@@ -44,3 +44,15 @@ def find_customer_revenue_percentiles(
               for percentile in percentiles]
 
     return values
+
+def find_most_common_traffic_sources(dataset):
+    """Find .
+
+    args:
+       dataset (Dataset): the google analytics dataset
+
+    returns:
+       Series of 5 most common traffic sources.
```

Comment:
(Optional) you could make this an argument, to allow the caller to specify the # of most common traffic sources